components/automate-ui/README.md

```diff
@@ -27,7 +27,7 @@ With the plugin installed, this TOC will update automatically when you save the
 
 ## Requirements
 
-1. node. We currently use node 8.9.4, but the current version number is always
+1. node. We currently use node 10.15.2, but the current version number is always
```

Comment:
While you're at it, the `8.9.4` version number appears a few lines later in the README, that should probably get updated as well as the language about the npm version. The NVM link is also outdated (it currently redirects but could be broken in the future).

Reply:
Thanks @phiggins - updated node and npm versions and updated the link as well.